api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/kpis_controller.rb

```diff
@@ -143,7 +140,7 @@ def kpi_parent
 
     def kpi_create_params
       whitelist = [:dashboard_id, :widget_id, :endpoint, :source, :element_watched, {extra_watchables: []}]
-      extract_params(whitelist)
+      extract_params(whitelist).merge(dashboard_id: params[:dashboard_id])
```

Comment:
yeah, it should not be there. The whitelist parameters are taking it from kpi inside the prams. dashboard_id is on the top level. (see `extract_params` method)